### DISCUSSION TITLE
Amazing theme for technical documentation

### DISCUSSION THREAD

**Discussion by dtomlinson91** [vuepress-theme-hope/vuepress-theme-hope#3538] [Q&A] [answered]
Just a big thank you for the work you've put into this theme.

We're using it for technical documentation for our team and it's a breeze to use, and was incredibly easy to get up and running. We will be using this for years.

<img width="1728" alt="SCR-20231013-uiwz-2" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/53234158/a3c1473b-7f3d-474e-be00-4121f09e4e64">

Do you have a donation link? I'd be happy to throw a contribution your way as a thank you for all the work.

**Comment by Mister-Hope** [comment, marked as answer]
the theme repo has a sponser button, and you can choose one
